pkgs/development/libraries/dconf/default.nix

```diff
@@ -30,27 +33,26 @@ stdenv.mkDerivation rec {
   nativeBuildInputs = [
     meson
     ninja
-    vala
     pkg-config
     python3
     libxslt
     libxml2
     glib
-    gtk-doc
     docbook-xsl-nons
     docbook_xml_dtd_42
-  ];
+  ] ++ stdenv.lib.optional (!isCross) gtk-doc;
```

Comment:
The docbook and lixml2/libxslt can be moved here as well.

Reply:
xsltproc and docbook are apprently still needed for man page generation. I was able to remove libxml2 entirely, and pushed it in a second commit on top.